### PR TITLE
Add command SMISMEMBER

### PR DESCRIPTION
**Pull request by nigelis**

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -200,6 +200,7 @@ type Cmdable interface {
 	SInter(ctx context.Context, keys ...string) *StringSliceCmd
 	SInterStore(ctx context.Context, destination string, keys ...string) *IntCmd
 	SIsMember(ctx context.Context, key string, member interface{}) *BoolCmd
+	SMIsMember(ctx context.Context, key string, members ...interface{}) *BoolSliceCmd
```
Comment (vmihailenco) [inline commands.go:203]: I believe we should change `SIsMember` to accept multiple args and use `smismember` if `len(members) > 1`. What do you think?

Comment (nigelis) [reply commands.go:203]: I understand that these two commands are quite similar, but changing `SIsMember` to accept multiple args breaks the API compatibility. The current type of return value is `*BoolCmd`, while the desired is `*BoolSliceCmd`.

Comment (monkey92t) [reply commands.go:203]: Can we consider modifying it in v9? they function exactly the same.

Comment (nigelis) [reply commands.go:203]: This looks good to me.

I can modify the `SIsMember` and open another PR to branch v9 after you draw the conclusion.

Comment (vmihailenco) [reply commands.go:203]: I haven't realized they return different results. No need to merge them then.

**Pull request merged**